src/crate/theme/rtd/crate/static/css/crateio-rtd.css

```diff
@@ -1,11 +1,3 @@
-.wrapper-content-right .section {
```

Comment:
why are we removing those? without them the headline starts like this (left is now, right is after removing those two entries):
![grafik](https://user-images.githubusercontent.com/8767096/114392257-cc4f1080-9b98-11eb-9435-d06b8deede4c.png)

Reply:
@amotl checked again, removing those two entries creates a lot of whitespace, probably better if we keep those, right?

Reply:
Yeah, our intention was actually to get rid of too much whitespace (on Desktop). Thank you for checking that on Mobile, maybe I was a bit too quick removing those CSS declarations. I will have another look.

Reply:
@amotl what should we do here, just keep the original .wrapper-content css? If we put them back in, I think this should be good to go (or do you want to make the general margin-top smaller?)

Reply:
I am currently working on this and, when trying to put them back in, it is still not right.

![image](https://user-images.githubusercontent.com/453543/119490353-08ab8a00-bd5d-11eb-9093-85792a9aada5.png)

Reply:
Fixed with 5928a6c.